src/mscorlib/src/System/TimeZoneInfo.cs

```diff
@@ -1420,7 +1422,7 @@ private class OffsetAndRule {
             return localConverted;           
         }
 
-
+#if FEATURE_WIN32_REGISTRY
```

Comment:
we need to make sure Linux flavor is not compiled with FEATURE_WIN32_REGISTRY :-)